src/reducers/ui.js

```diff
@@ -14,20 +14,14 @@ import { prefs } from "../utils/prefs";
 import type { Action, panelPositionType } from "../actions/types";
 import type { Record } from "../utils/makeRecord";
 
-export type SymbolSearchType = "functions" | "variables";
-export type ActiveSearchType =
-  | "project"
-  | "source"
-  | "file"
-  | "symbol"
-  | "line";
+export type ActiveSearchType = "project" | "file";
 
 export type OrientationType = "horizontal" | "vertical";
 
 export type UIState = {
   activeSearch: ?ActiveSearchType,
+  quickOpenEnabled: boolean,
```

Comment:
hmm, it might be nice to move this to the `quickOpen` reducer.

Reply:
I had waffled on that a bit. I had considered it for a bit but figured I could move it. Since it was whether it appeared or not it seemed UIish but it was the quick open's state heh.